pkgs/development/libraries/glpk/default.nix

```diff
@@ -72,7 +73,7 @@ stdenv.mkDerivation rec {
     homepage = "https://www.gnu.org/software/glpk/";
     license = stdenv.lib.licenses.gpl3Plus;
 
-    maintainers = with stdenv.lib.maintainers; [ bjg timokau ];
+    maintainers = with lib.maintainers; [ bjg ] ++ teams.sage.members;
```

Comment:
```
  meta = with stdenv.lib; {
    description = "The GNU Linear Programming Kit";
    longDescription =
      '' The GNU Linear Programming Kit is intended for solving large
         scale linear programming problems by means of the revised
         simplex method.  It is a set of routines written in the ANSI C
         programming language and organized in the form of a library.
      '';
    homepage = "https://www.gnu.org/software/glpk/";
    license = stdenv.lib.licenses.gpl3Plus;
    maintainers = with maintainers; [ bjg ] ++ teams.sage.members;
    platforms = platforms.all;
  };
```

This fixes the current eval error. Also the change in line two where you added lib can be reverted with this.

Reply:
Oh, I had assumed `teams` was an attribute of `maintainers`, not of `lib` directly. Thanks!